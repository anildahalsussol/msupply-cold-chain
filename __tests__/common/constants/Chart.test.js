import { CHART } from '~constants';

describe('Constants: CHART', () => {
  it('Matches snapshot', () => {
    expect(CHART).toMatchSnapshot();
  });
});
