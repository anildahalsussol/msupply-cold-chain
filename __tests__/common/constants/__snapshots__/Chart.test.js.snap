// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Constants: CHART Matches snapshot 1`] = `
Object {
  "2": "X_DOMAIN_PADDING",
  "20": "Y_DOMAIN_PADDING",
  "5": "NUMBER_OF_TICKS_Y",
  "INTERPOLATION": "natural",
  "LINE_GRADIENT_ID": "url(#grad)",
  "NUMBER_OF_TICKS_Y": 5,
  "X_DOMAIN_PADDING": 2,
  "X_KEY": "timestamp",
  "Y_DOMAIN_PADDING": 20,
  "Y_KEY": "temperature",
}
`;
